lib/real.js

```diff
@@ -89,9 +90,79 @@ SecureWorker.getQuoteData = function getQuoteData(quote) {
 
 // Do a remote attestation of a given quote. Quote can come from this or some other machine.
 // Returns a signed remote attestation statement on successful remote attestation.
-// Inputs and outputs are an arraybuffer.
-SecureWorker.getRemoteAttestation = function getRemoteAttestation(quote) {
-  throw new Error("Not implemented.");
+// Input is an arraybuffer containing the quote, Subscription Key and a callback function structured at function(result,err)
+// The RA signature is returned using the callback function as an arraybuffer
+// If an error occurs, result will be undefined and err will contain a string
+SecureWorker.getRemoteAttestation = function getRemoteAttestation(quote, subscription_key, callbackFn) {
+  var self=this;
+  
+  var quote_data = self.getQuoteData(quote);
+  var quote_str = String.fromCharCode.apply(null,new UInt16Array(quote_data));
+
+  var ra_url = "https://api.trustedservices.intel.com/sgx/attestation/v3/report";
```

Comment:
SGX now also allows non-Intel attestation servers, so this could also be an option to the call?

Reply:
Definitely, I'll add to the next commit

Reply:
See: https://software.intel.com/en-us/blogs/2018/12/09/an-update-on-3rd-party-attestation